lib/activitylog/normalise-comments.js

```diff
@@ -0,0 +1,37 @@
+module.exports = activityLog => {
+  if (!activityLog.length) {
+    return activityLog;
+  }
+
+  const deletedCommentIds = activityLog.filter(log => log.eventName === 'delete-comment').map(log => log.event.meta.id);
+
+  const updatedComments = activityLog
+    .filter(log => log.eventName === 'update-comment')
+    .sort((a, b) => a.updatedAt <= b.updatedAt ? 1 : -1); // most recent is first
```

Comment:
I think the activity log is already sorted newest to oldest so we might not need this...